### ISSUE TITLE
Custom font not working on the simplest app ever ... 

### ISSUE THREAD

**Issue by Name-less** [chrisjenx/Calligraphy#252]
Hello, I tried to use your custom font in my project without success.

I deleted a lot of things on your sample project just to get the sample I needed. It worked on your sample project. But when I create a new hello word project, add the lib and some style, nothing happen ... (the style color is taken but not the font).

Do you have any idea ? 

Again, this simple architecture project is perfectly working on your Sample app cleaned of all the stuff I do not want to use for the moment ...

Here is my simple app : 

(My theme AppTheme is correctly set on the Manifest, and the font under the assets/fonts folder)

MainActivity.java (start activity) : 
```
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    @Override
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase));
    }
}

```
styles.xml : 

```
<resources
    xmlns:tools="http://schemas.android.com/tools"
    tools:ignore="NewApi">

    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">

        <item name="android:textViewStyle">@style/AppTheme.Widget.TextView</item>

    </style>

    <style name="AppTheme.Widget.TextView" parent="android:Widget.Holo.Light.TextView">
        <!-- Custom Attr-->
        <item name="fontPath">fonts/gtw.ttf</item>
        <item name="android:textColor">#457654</item>
    </style>

</resources>
```

activity_main.xml : 

```
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <TextView
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:text="et au tu va marhcer un jour toi ?"/>

</RelativeLayout>

```

And of course my dependencies : 

```
dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    testCompile 'junit:junit:4.12'
    compile 'com.android.support:appcompat-v7:23.1.1'
    compile 'uk.co.chrisjenx:calligraphy:2.1.0'
}
```

**Comment by chrisjenx** [comment]
Is the new project using BuildTools 2.x+ or instant run?

**Comment by Name-less** [comment]
I am not currently using android studio 2.0 so I am not using instant run ... I the point I am missing ? I succes to use the library using fontPath on some xml files in an other app but not in the one above. Nice library btw !

**Comment by chrisjenx** [comment]
Are you able to zip up the project? I am interested to see what's going on.

**Comment by Name-less** [comment]
Yep sure, here he is !

[CaligraphyTest.zip](https://github.com/chrisjenx/Calligraphy/files/148121/CaligraphyTest.zip)

**Comment by chrisjenx** [comment]
@Name-less, OK, I'm baffled too, I've got my sample to match the one in you're sample to no luck. 

I'll play a little more but this is bizzare.

**Comment by Name-less** [comment]
Thank you, tell me if you find something !

**Comment by chrisjenx** [comment]
I put a fix in for the AppCompat classes which is in the latest Calligraphy
snapshot. That's why default themes don't work with the current 2.1.0.

I'll see about releasing the snapshot.

**Comment by chrisjenx** [comment]
Try the `2.2.0-SNAPSHOT`

**Comment by Name-less** [comment]
Thanks, I will try, but I am not able to get the latest Snapshot using gradle, how can I do that ?

Here is my gradle conf : 

```
configurations.all {
    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
}

dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    testCompile 'junit:junit:4.12'
    compile 'com.android.support:appcompat-v7:23.2.0'
    compile 'uk.co.chrisjenx:calligraphy:latest.integration'
}
```

**Comment by chrisjenx** [comment]
Check:
https://github.com/chrisjenx/Calligraphy/issues/185#issuecomment-128170474

**Comment by chrisjenx** [comment]
2.2.0 released, this should resolve now.

**Comment by goodlifechris** [comment]
Thanks Chris for this.  I had an issue with the latest which was 2.3.0 

Error:Failed to resolve: com.android.support:appcompat-v7:25.3.1
<a href="install.m2.repo">Install Repository and sync project</a><br><a href="../app/build.gradle">Open File</a><br><a href="open.dependency.in.project.structure">Show in Project Structure dialog</a>

Error:Failed to resolve: annotationProcessor
<a href="../app/build.gradle">Open File</a>

Resolved this isses with 2.2.0 Thankyou once again
